Impossible-Finance-Agents/New-Pair/package.json

```
@@ -16,18 +20,20 @@
     "disable": "forta-agent disable",
     "enable": "forta-agent enable",
     "keyfile": "forta-agent keyfile",
-    "test": "jest"
+    "test": "jest",
+    "format": "prettier --write \"src/**/*.ts\""
   },
   "dependencies": {
-    "forta-agent": "^0.0.34",
-    "forta-agent-tools": "^1.0.32"
+    "forta-agent": "^0.1.3"  
   },
   "devDependencies": {
     "@types/jest": "^27.0.1",
     "@types/nodemon": "^1.19.0",
+    "forta-agent-tools": "^2.0.5",
```

```suggestion
    "forta-agent-tools": "^2.0.7",
```
Same as `forta-agent`